taqueria-sdk/index.ts

```diff
@@ -233,7 +233,8 @@ const formatArgs = (args: Record<string, string>) => {
 	const entries = Object.entries(args).map(
 		([key, value]) => {
 			if (key === 'config') return [key, JSON.parse(value)];
-			if (value === 'false' || value === 'true') return [key, Boolean(value)];
+			else if (value === 'false') return [key, false];
+			else if (value === 'true') return [key, true];
```

Comment:
Isn't
Boolean('false') the same as false
And 
Isn't Boolean('true') the same as true
?

Reply:
Nope - I would have thought so too. Hence the change/fix.

Reply:
That is really quite disconcering.